quick-start/src/main/ui/app/harmonize-flow-options/harmonize-flow-options.component.ts

```diff
@@ -104,7 +104,7 @@ export class HarmonizeFlowOptionsComponent implements OnInit, OnChanges {
         // Temporarily saving locally
         localStorage.removeItem(storedMap);
         this.mapName = null;
-        // this.mapService.deleteMap(this.flow.entityName, storedMap);  // TODO: Restore once backend exists
+        this.mapService.deleteMap(storedMap);  // TODO: Restore once backend exists
```

Comment:
you left the comment in place

Reply:
Don't pull this till I give the go ahead, changing things!